src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -17,7 +17,7 @@ import IconMail from '../../icons/general/IconMail';
 import IconClock from '../../icons/general/IconClock';
 import IconGlobe from '../../icons/general/IconGlobe';
 import { amaranth } from '../../styles/variables';
-import type { itemType as ItemType } from '../../common/box-types';
+import type { itemType as ItemType } from '../../common/flowTypes';
```

Comment:
can just make it ItemType inside the file?

Reply:
+1 to uppercasing all types